app/templates/ui/styles/main.less

```diff
@@ -67,6 +66,33 @@ nav.nav {
   padding-top: 15px;
 }
 
+.chiclet {
+  margin: 3px;
+}
+
+/* negated and added chiclets */
+.chiclet  {
+    position: relative;
+}
+/* set up the added one first */
+.chiclet:before {
+    content: "\f055";
+    font-family: FontAwesome;
```

Comment:
This is a less file, right? Can't you pull in fa classes here? `.chiclet { .fa; .fa-whatever; }`?